### PR TITLE
Update changelog for release

### PR DESCRIPTION
**Pull request by cijothomas**
Following the pattern described in https://github.com/open-telemetry/opentelemetry-dotnet/issues/2991#issuecomment-1083242602

### CODE REVIEW

**Review by reyang — CHANGES_REQUESTED** [review]
https://github.com/open-telemetry/opentelemetry-dotnet/pull/3108/files#r838685537

File: src/OpenTelemetry.Instrumentation.StackExchangeRedis/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+## 1.0.0-rc9.1
```
[the remark this answers is not in the document]
Comment (cijothomas) [reply src/OpenTelemetry.Instrumentation.StackExchangeRedis/CHANGELOG.md:5]: rc10.1 will still be considered older than rc9, so we need to use something which will be newer than rc9. Picking rc9.1, rc9.2, and so on for this.
(rc10 will be unlisted from nuget)

[the remark this answers is not in the document]
Comment (reyang) [reply src/OpenTelemetry.Instrumentation.StackExchangeRedis/CHANGELOG.md:5]: yikes!!!

**Review by reyang — APPROVED** [review]

File: src/OpenTelemetry.Instrumentation.StackExchangeRedis/CHANGELOG.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (reyang) [reply src/OpenTelemetry.Instrumentation.StackExchangeRedis/CHANGELOG.md:5]: Perhaps tweak the change log:

```markdown
## 1.0.0-rc10 (broken)
```

[the remark this answers is not in the document]
Comment (cijothomas) [reply src/OpenTelemetry.Instrumentation.StackExchangeRedis/CHANGELOG.md:5]: Good point. Will do

File: src/OpenTelemetry.Instrumentation.AspNetCore/CHANGELOG.md [hunk 2]
```diff
@@ -10,7 +10,11 @@
 
 ## Unreleased
 
-## 1.0.0-rc10
+## 1.0.0-rc9.1
```
[the remark this answers is not in the document]
Comment (reyang) [reply src/OpenTelemetry.Instrumentation.AspNetCore/CHANGELOG.md:13]: How about `1.0.0-rc31415926535897932384626...` similar to what [TeX](https://en.wikipedia.org/wiki/TeX#:~:text=The%20current%20version%20of%20TeX,was%20last%20updated%20in%202021.) did.

**Pull request merged**